platform_versioned_docs/version-24.2/compute-envs/aws-batch.mdx

```diff
@@ -247,7 +249,7 @@ Specify the **Allocation strategy** and indicate any preferred **Instance types*
 - Add an execution role ARN to the **Batch execution role** field to grant permissions to make API calls on your behalf to the ECS container used by Batch. This is required if the pipeline launched with this compute environment needs access to the secrets stored in this workspace. This field can be ignored if you are not using secrets.
 - Specify an EBS block size (in GB) in the **EBS auto-expandable block size** field to control the initial size of the EBS auto-expandable volume. New blocks of this size are added when the volume begins to run out of free space. This feature is deprecated, and is not compatible with Fusion v2.
 - Enter the **Boot disk size** (in GB) to specify the size of the boot disk in the VMs created by this compute environment.
-- If you're using **Spot** instances, you can also specify the **Cost percentage**, which is the maximum allowed price of a **Spot** instance as a percentage of the **On-Demand** price for that instance type. Spot instances will not be launched until the current spot price is below the specified cost percentage.
+- If you're using **Spot** Instances, you can also specify the **Cost percentage**, which is the maximum allowed price of a **Spot** Instance as a percentage of the **On-Demand** price for that instance type. Spot Instances will not be launched until the current Spot price is below the specified cost percentage.
```

Comment:
![image](https://github.com/user-attachments/assets/10e5dca0-4814-4a70-9bd4-ff0857d88cea)
AWS does not capitalize the word instance in their docs, so we shouldn't capitalize it in ours.